frontend/src/components/Collections/components/CollectionsGrid/components/CollectionRow/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { Intent, Tag } from "@blueprintjs/core";
+import loadable from "@loadable/component";
+import React, { FC } from "react";
+import {
+  ACCESS_TYPE,
+  COLLECTION_LINK_TYPE,
+  VISIBILITY_TYPE,
+} from "src/common/entities";
+import { useCollection } from "src/common/queries/collections";
+import { StyledCell } from "../common/style";
+import { LeftAlignedDetailsCell } from "./components/common/style";
+import { aggregateDatasetsMetadata } from "./components/common/utils";
+import {
+  CollectionTitleText,
+  RightAlignedDetailsCell,
+  StyledRow,
+} from "./style";
+
+interface Props {
+  id: string;
+  accessType: ACCESS_TYPE;
+  visibility: VISIBILITY_TYPE;
+}
+
+const AsyncPopover = loadable(
+  () =>
+    /*webpackChunkName: 'CollectionRow/components/Popover' */ import(
+      "src/components/Collections/components/CollectionsGrid/components/CollectionRow/components/Popover"
+    )
+);
+
+const conditionalPopover = (values: string[]) => {
+  if (!values || values.length === 0) {
+    return <LeftAlignedDetailsCell>-</LeftAlignedDetailsCell>;
+  }
+
+  return <AsyncPopover values={values} />;
+};
+
+type DOI = {
+  doi: string;
+  link: string;
+};
+
+const CollectionRow: FC<Props> = (props) => {
+  const { data: collection } = useCollection(props.id, props.visibility);
+
+  if (!collection) return null;
+
+  // If there is an explicity accessType only show collections with that accessType
```

Comment:
This is commented out for now to show what it looks like to have both public and private collections in My Collections page, because a user can have `WRITE` access to both public and private collections that they own. But right now there's no such user in our database!

Before merging, we need to restore this back to:

```tsx
// If there is an explicity accessType only show collections with that accessType
if (props.accessType && collection.access_type !== props.accessType) {
  return null;
}
```